dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -116,3 +116,19 @@ object reflects the newly requested size in `PersistentVolume.Spec.Capacity`. At
 this point, you can create or re-create a new pod from the PVC to finish the
 file system resizing. Once the pod is running, the newly requested size is
 available and `FileSystemResizePending` condition is removed from the PVC.
+
+[[recover_from_resize_failure]]
+== Recover from failure in Expanding volumes
+
+If for some reason expanding underlying storage fails either on master or on node,
```

Comment:
s/If for some reason expanding underlying storage fails either on master or on node/If expanding underlying storage fails either on master or node,